pages/common/rspec.md

```diff
@@ -0,0 +1,32 @@
+# rspec
+
+> RSpec is a behavior-driven development testing framework written in programming language Ruby to test Ruby code.
```

Comment:
Perhaps this will read better?
```suggestion
> Behavior-driven development testing framework written in Ruby to test Ruby code.
```

Reply:
Thanks @sbrl, no it's not a fuzzing test framework, but Rspec runs the tests in randomised order by default, `--seed` allows re-running the tests with the same order if you need to :)